### DISCUSSION TITLE
iframe doest display by connection refused from remote server

### DISCUSSION THREAD

**Discussion by jay-jjwu** [frangoteam/FUXA#876] [Q&A]
Hi, 
Why i get an error of "connection refused by setup inframe link to a remote server",?
See screenshots,....On the left, it's local view page, and it works well.
But when i link to a remote server via http, it comes an error of connection refused.
Please help, thanks!

![image](https://github.com/frangoteam/FUXA/assets/78014320/7cfc152c-8379-4c82-9ac0-d1f8edfa5cbf)

**Comment by unocelli** [comment]
Hi, this feature still needs to be implemented.
